app/screens/AppNavigator/screens/Settings/screens/NetworkDetails.tsx

```diff
@@ -0,0 +1,134 @@
+import { useNetworkContext } from '@contexts/NetworkContext'
+import { RootState } from '@store'
+import React from 'react'
+import dayjs from 'dayjs'
+import { useSelector } from 'react-redux'
+import { View } from '../../../../../components'
+import { ThemedText, ThemedSectionTitle, ThemedSectionList, ThemedScrollView, ThemedView } from '../../../../../components/themed'
+import { tailwind } from '../../../../../tailwind'
+import { translate } from '../../../../../translations'
+
+enum NETWORK {
+  ConnectedStatus = 'YOU ARE CURRENTLY CONNECTED TO',
+  NetworkDetails = 'NETWORK DETAILS'
+}
+
+interface StatItemProps {
+  label: string
+  status: string
+  statusIcon?: JSX.Element
+  testID: string
+}
+
+export function NetworkDetails (): JSX.Element {
+  const { network } = useNetworkContext()
+  const { connected, count: blockCount, masternodeCount, lastSync } = useSelector((state: RootState) => state.block)
+  const syncFormattedDate = (lastSync != null) ? dayjs(lastSync).format('MMM D, h:mm a') : ''
+
+  const statsData = [
+    {
+      key: NETWORK.ConnectedStatus,
```

Comment:
ConnectedStatus doesn't look aligned on mobile

Reply:
![image](https://user-images.githubusercontent.com/25013382/133594825-dc75be0c-b41d-4fc7-948b-792eefa9d7a4.png)

The green dot should be aligned in the middle of Connected

Reply:
and maybe increase the size a bit

Reply:
Will make the changes